ogr/services/github/service.py

```diff
@@ -60,6 +66,21 @@ def __init__(
         self.github = github.Github(login_or_token=self.token)
         self.read_only = read_only
 
+        self.tokman_instance_url = tokman_instance_url
+        self._token_manager = token_manager
+
+        self.__check_for_multiple_auths(
+            (token, github_app_id, tokman_instance_url, token_manager)
+        )
```

Comment:
What about hiding all attributes related to the authentication into the token manager(s)?

Reply:
So have a method that will receive all auth-related attributes and return an instance of token manager?